libfwupdplugin/fu-volume.c

```diff
@@ -440,7 +440,14 @@ fu_volume_get_block_size(FuVolume *self, GError **error)
 	g_autoptr(GVariant) val = NULL;
 
 	g_return_val_if_fail(FU_IS_VOLUME(self), 0);
-	g_return_val_if_fail(G_IS_DBUS_PROXY(self->proxy_blk), 0);
+
+	if (!G_IS_DBUS_PROXY(self->proxy_blk)) {
```

Comment:
isn't `self->proxy_blk == NULL` simpler?

Reply:
Yeah I'll switch to that.